packages/scratch-gui/test/unit/components/button.test.jsx

```diff
@@ -1,23 +1,27 @@
 import React from 'react';
-import {shallow} from 'enzyme';
+import { render, fireEvent } from '@testing-library/react';
+import '@testing-library/jest-dom';
 import ButtonComponent from '../../../src/components/button/button';
-import renderer from 'react-test-renderer';
 
 describe('ButtonComponent', () => {
     test('matches snapshot', () => {
-        const onClick = jest.fn();
-        const component = renderer.create(
-            <ButtonComponent onClick={onClick} />
+        const { container } = render(
+            <ButtonComponent onClick={jest.fn()} />
```

Comment:
At some places it is used `const onClick = jest.fn();` and at others `jest.fn()` is passed directly to onClick. We should unify the syntax everywhere

Reply:
In this case it may be irrelevant as we compare the component to a snapshot